### PR TITLE
more instructions for ios

### PR DESCRIPTION
**Pull request by sethladd**
* shift docs around
* reorder getting started

### CODE REVIEW

**Comment by sethladd** [comment]
cc @chinmaygarde and @abarth

File: building-apk.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+layout: page
+title: Building an APK for Android
+permalink: /building-apk/
+---
+
+_Note: An `apk/AndroidManifest.xml` file was already created for
+your app by `flutter init`. You can check it out if you like,
```
Comment (eseidelGoogle) [inline building-apk.md]: Be aware of https://github.com/flutter/flutter/issues/1006.  We have other bugs on file about how this shouldn't just be for apks and how we should include an icon in the template for both ios and android.

File: building-apk.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+but you don't need to modify it
+when getting started._
+
+First, ensure you installed the Android SDK tools
+(see "Setting up your Android device" from [Getting started](getting-started)).
+
+Run `android update sdk` from the command line,
+and install SDK version 22 and build tools version 22.0.1.
+For now, Flutter currently expects only _these_ versions.
```
Comment (eseidelGoogle) [inline building-apk.md]: Are we tied to the Android version or the adb version?  How do I check my android version?  Maybe these are things the proposed `flutter doctor` should do for me.

File: building-apk.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+Then, inside your Flutter app, run `flutter apk`.
+
+The APK can be found in `$MY_FLUTTER_APP/build/app.apk`.
```
Comment (eseidelGoogle) [inline building-apk.md]: IMO eventually we should change this to build/$appname.apk, but yeah.

File: getting-started.md [hunk 4]
```diff
@@ -153,6 +143,38 @@ The `logs` command lets you see textual output from your app, including `print`
 statements and unhandled exceptions. To avoid confusion from old log messages,
 you might want to use `flutter logs --clear` to clear the logs between runs.
 
+## Setting up your iOS device or simulator
+
+You must use a Mac, with Xcode installed, to develop iOS apps with Flutter.
```
Comment (eseidelGoogle) [inline getting-started.md]: Xcode 7+?

File: getting-started.md [hunk 5]
```diff
@@ -153,6 +143,38 @@ The `logs` command lets you see textual output from your app, including `print`
...
+to add iOS configuration files.
+
+You can shut down XCode, if it is started automatically.
+
+## Running your app in the iOS simulator
+
+- Run `Simulator.app`
+  - You can find Simulator.app in
+_Applications/Xcode/Contents/Developer/Applications_ or via _Spotlight_
+  - To ensure your simulator uses a device type of iPhone 5 or above,
```
Comment (eseidelGoogle) [inline getting-started.md]: Specifically it's that we only support 64-bit devices at this time.  Once I have our new infra up and running we'll support both.

**Comment by eseidelGoogle** [comment]
lgtm.  These will need more updates as we fix more bugs.

File: getting-started.md (hunk 4 above)
Comment (sethladd) [inline getting-started.md]: Done

**Pull request merged**